roles/telemetry_logging/tasks/main.yml

```diff
@@ -0,0 +1,500 @@
+      loop: "{{ packagemanifests_list }}"
+
+
+- name: Verify openshift-operators-redhat subscriptions exist
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+    nspace: openshift-operators-redhat 
+    subscript_out_file: verify_openshift-operators-redhat_subscriptions_lresults.log 
+    subscription_list:
+      - loki-operator
+
+  tasks:
+    - name: Remove "{{ subscript_out_file }}"
+      ansible.builtin.file:
+        path: "{{ subscript_out_file }}"
+        state: absent
+      changed_when: false
+
+    - name: Run packagemanifest tests
+      ansible.builtin.include_tasks: logging_subscription_tests.yml
+      loop: "{{ subscription_list }}"
+
+
+- name: Verify ctlplane node containers
+  hosts: ctlplane_nodes
+  gather_facts: no
+  vars:
+    container_out_file: "verify_ctlplane_node_containers_{{ inventory_hostname }}_lresults.log"
+    container_list:
+      - ceilometer_agent_compute
+      - ceilometer_agent_ipmi
+      - node_exporter
+
+  tasks:
+    - name: Remove "{{ container_out_file }}"
+      ansible.builtin.file:
+        path: "{{ container_out_file }}"
+        state: absent
+      changed_when: false
+
+    - name: Verify containers on edpm-compute nodes
+      ansible.builtin.include_tasks: logging_container_tests.yml
+      loop: "{{ container_list }}"
+
+    - name: Copy output file to hypervisor
+      ansible.builtin.fetch:
+        src: "/root/{{ container_out_file }}"
+        dest: ./
+        flat: true
+      changed_when: false
+
+
+- name: Verify ctlplane node files
+  hosts: ctlplane_nodes
+  gather_facts: no
+  vars:
+    file_exists_outfile: "verify_ctlplane_node_files_{{ inventory_hostname }}_lresults.log"
+    file_list:
+      - /etc/rsyslog.d/10-telemetry.conf
+
+  tasks:
+    - name: Remove "{{ file_exists_outfile }}"
+      ansible.builtin.file:
+        path: "{{ file_exists_outfile }}"
+        state: absent
+      changed_when: false
+
+    - name: Check for Files
+      ansible.builtin.include_tasks: logging_file_tests.yml
+      loop: "{{ file_list }}"
+
+    - name: Copy output file to hypervisor
+      ansible.builtin.fetch:
+        src: "/root/{{ file_exists_outfile }}"
+        dest: ./
+        flat: true
+      changed_when: false
+
+
+- name: Verify journalctl identifiers
+  hosts: ctlplane_nodes
+  gather_facts: no
+  vars:
+    journal_exists_outfile: "verify_journalctl_identifiers_{{ inventory_hostname }}_lresults.log"
+    identifiers_list:
+      - ceilometer_agent_compute
+      - nova_compute
+
+  tasks:
+    - name: Remove "{{ journal_exists_outfile }}"
+      ansible.builtin.file:
+        path: "{{ journal_exists_outfile }}"
+        state: absent
+      changed_when: false
+
+    - name: Verify journalctl identifiers 
+      ansible.builtin.include_tasks: logging_journal_tests.yml
+      loop: "{{ identifiers_list }}"
+
+    - name: Copy output file to hypervisor
+      ansible.builtin.fetch:
+        src: "/root/{{ journal_exists_outfile }}"
```

Comment:
This file path will need to be changed.

I assume it's in /root because you're using `become: true` and the file ends up in the home directory.

You can replace `/root...` with `"{{ ansible_env.HOME }}/{{ journal_exists_outfile }}"`

However, in general, it would be more useful to define a log_output_dir, so you can configure that directory more easily.

I think that this outputfile  may become obsolete once we switch to using the custom_logger for running the telemetry_logging role.